cparser/regexParser.js

```diff
@@ -260,8 +184,8 @@ module.exports = that = {
 			// Comments
 			"|(/\\*[^*]*(?:\\*(?!/)[^*]*)*\\*/)|(//.*?$)",
 
-			// Pre-processor
-			"|(#(?:\\\\\\n|.)*)"
+			// Include statements
+			"|(#include.+$)"
```

Comment:
This is the big question. We should discuss more in Slack since you are not convinced adding application.h after the includes is a good compromise.